### ISSUE TITLE
Wrong file names shown in tracebacks since py2exe 0.11.0.0

### ISSUE THREAD

**Issue by lukaszgo1** [py2exe/py2exe#114]
Compile and start 
[fnames_in_tracebacks.zip](https://github.com/py2exe/py2exe/files/7545650/fnames_in_tracebacks.zip)
With version 0.11.0.0 the following is shown:
```
Traceback (most recent call last):
  File "D:\fnames_in_tracebacks\issue.py", line 4, in <module>
    raise RuntimeError("sss")
RuntimeError: sss

```

whereas previously we got:
```
Traceback (most recent call last):
  File "issue.pyc", line 4, in <module>
    raise RuntimeError("sss")
RuntimeError: sss

```

This seems similar to the problem solved in #12

**Comment by albertosottile** [comment]
Seems like I forgot this in the new module finder, nice catch. Could you try the wheels just built by Actions with the patch? Thanks https://github.com/py2exe/py2exe/actions/runs/1468362964

**Comment by lukaszgo1** [comment]
The wheels linked above solve the problem both for my  MWE and in NVDA. Are you planning to create a official release with this fix?

**Comment by albertosottile** [comment]
Yes, but not immediately? I would like to wait for a little if anything else pops up from users before releasing the first patch. Since this is only a cosmetic issue, I would imagine you can proceed with your tests with 0.11.0.0. However, if NVDA has an immediate need for this fix, I can release it now.

**Comment by lukaszgo1** [comment]
My attempt to update NVDA to 0.11.0.0 was reverted due to this and #113, so if possible I would appreciate a release if this is not too much trouble.

**Comment by albertosottile** [comment]
I was hoping that only #113 was the blocking issue... Nevertheless, I just released `0.11.0.1` with the fix for this bug on PyPI. Hope this helps.
